react-components/src/utilities/isSameModel.ts

```diff
@@ -52,13 +50,24 @@ export function isSamePointCloudModel(
   return (
     model0.modelId === model1.modelId &&
     model0.revisionId === model1.revisionId &&
-    (model0.transform === model1.transform ||
-      (model0.transform !== undefined &&
-        model1.transform !== undefined &&
-        model0.transform.equals(model1.transform)))
+    isSameTransform(model0.transform, model1.transform)
   );
 }
 
+const identity = new Matrix4();
+
+function isSameTransform(m0: Matrix4 | undefined, m1: Matrix4 | undefined): boolean {
+  if ((m0 === undefined || m0.equals(identity)) && (m1 === undefined || m1.equals(identity))) {
+    return true;
+  }
```

Comment:
m0 being undefined and m1 being a matrix4 identity is not really the same transform, for instance. Or something I've forgot?

Reply:
Ok so based on our discussion, the idea is to either being one undefined and another one is identity, it will be treated as with the same transform later on, so it is intentional. Ok.